src/main/java/com/hubspot/jinjava/lib/filter/DateTimeFormatFilter.java

```diff
@@ -27,10 +31,19 @@ public String getName() {
 
   @Override
   public Object filter(Object var, JinjavaInterpreter interpreter,
-      String... args) {
+      Object... args) {
+
+    List<String> stringArgs = new ArrayList<>();
```

Comment:
You could do this all in one loop. In fact, you don't need `stringArgs` at all. I know this was just moved from another file, but might as well clean it up while we're in here.

Reply:
done :+1: